### ISSUE TITLE
Upload quick2wire-python-api to pypi.

### ISSUE THREAD

**Issue by OrangeTux** [romilly/quick2wire-python-api#54]

**Comment by Peque** [comment]
@OrangeTux I know this issue was opened long time ago but, if you are still interested (or someone ends up here), I published the package in PyPI:

https://pypi.python.org/pypi/quick2wire-api

**Comment by joseangeljimenez** [comment]
Thanks for your work!

**Comment by joseangeljimenez** [comment]
By the way, maybe you should use the standard version system of Python packages x.y.z. Latest version you have published is 0.0.0.2, it should be 0.0.2, IMHO.

**Comment by Peque** [comment]
@joseangeljimenez I agree with you, but that is something @npryce-springer should change (I do not want to publish a version in PyPI that does not correspond to the actual version tag in the repository). :sweat_smile:
